client/my-sites/themes/controller.js

```diff
@@ -113,17 +113,30 @@ export function loggedOut( context, next ) {
 	next();
 }
 
+function getDetailsProps( context ) {
+	const { slug } = context.params;
+	const basePath = '/themes/:slug';
+	const analyticsPageTitle = 'Themes > Details Sheet';
+
+	const runClientAnalytics = function() {
+		analytics.pageView.record( basePath, analyticsPageTitle );
+	};
+
+	return {
+		themeSlug: slug,
+		title: buildTitle(
+			i18n.translate( 'Theme Details', { textOnly: true } )
```

Comment:
While we're here, we might think about what page title we want to give to theme sheets.